var/spack/repos/builtin/packages/py-pyprecice/package.py

```diff
@@ -53,4 +60,5 @@ def build_ext_args(self, spec, prefix):
         ]
 
     def install(self, spec, prefix):
-        self.setup_py("install", "--prefix={0}".format(prefix))
+        if self.version <= Version("2.1.1.1"):
+            self.setup_py("install", "--prefix={0}".format(prefix))
```

Comment:
Does the default install method not work for some reason?

Reply:
We had to use the non-default method due to https://github.com/precice/python-bindings/issues/64 in older versions. In Version `2.2.0.1` this issue is fixed, but we still have to use the non-default method for compatibility reasons. Or is there a better way how to do this in spack?

Here is the discussion from our original PR: https://github.com/spack/spack/pull/19558#discussion_r513123239.